src/simtools/camera/single_photon_electron_spectrum.py

```diff
@@ -123,7 +146,7 @@ def _derive_spectrum_norm_spe(self):
         if tmp_ap_file:
             command.extend(["-a", f"{tmp_ap_file.name}"])
             command.extend(["-s", f"{self.args_dict.get('scale_afterpulse_spectrum', 1.0)}"])
```

Comment:
Same comment as in the other PR about the default value.

Reply:
Fixed there.